aardvark/manage.py

```diff
@@ -300,51 +232,54 @@ def _prep_accounts(account_names):
     Returns a list of account numbers
     """
     matching_accounts = list()
-    account_names = account_names.split(',')
+    account_names = account_names.split(",")
     account_names = {name.lower().strip() for name in account_names}
 
     # create a new copy of the account_names list so we can remove accounts as needed
     for account in list(account_names):
-        if re.match('\d{12}', account):
+        if re.match("\d{12}", account):
             account_names.remove(account)
             matching_accounts.append(account)
 
     if not account_names:
         return matching_accounts
 
     try:
-        current_app.logger.info('getting bucket {}'.format(
-                                current_app.config.get('SWAG_BUCKET')))
-
-        swag = SWAGManager(**parse_swag_config_options(current_app.config.get('SWAG_OPTS')))
+        swag_opts = CONFIG["swag"]["opts"].get()
+        swag = SWAGManager(**swag_opts)
 
-        all_accounts = swag.get_all(current_app.config.get('SWAG_FILTER'))
+        all_accounts: List[Dict] = swag.get_all(CONFIG["swag"]["filter"].get())
```

Comment:
A little weird to see one type hint surrounded by things that don't have type hints

Reply:
True. My approach is "add in type hints as we touch things" to start moving toward better type hint coverage.